src/app-logic/published-profiles-store.js

```diff
@@ -97,12 +86,22 @@ async function open(): Promise<Database> {
       // to change this database format (and increased the version number)
       // and then downgraded to a version of profiler.firefox.com without those
       // changes.
-      // We delete the database and try again.
-      await deleteDB(DATABASE_NAME);
-      db = await reallyOpen();
-    } else {
-      throw e;
+      // Let's explain that in an error, that will be output to the console by
+      // the caller.
+      (window: any).deleteDB = () => deleteDB(DATABASE_NAME);
+      throw new Error(stripIndent`
+        We tried to open an existing published profiles store database with a
+        smaller version than the current one. We can't do that with IndexedDB.
+        The only way to recover is to delete the database and create a new one,
+        but we don't want to do this automatically.
+        Until this is fixed we won't be able to store newly published profiles
+        or retrieve previously published profiles.
+
+        If you want to delete the database, you can do so by running 'deleteDB()'
```

Comment:
Thanks for doing this!

Reply:
No action needed, just musing and over-engineering: It would be funny to throw an error with a react rendered component attached to it that could have a button that would do the `deleteDB` automatically.

Reply:
I think we could do it in eg `/my-profiles` when we display the full list, but in the main UI this is hard. I'd rather have an action button in a error notification :)